src/askem_beaker/contexts/mira_model_edit/context.py

```diff
@@ -117,45 +118,40 @@ async def reset_request(self, message):
         await self.send_mira_preview_message(parent_header=message.header)
 
     @intercept()
-    async def replace_template_name(self, message):
+    async def replace_template_name_request(self, message):
         content = message.content
 
         model = content.get("model")
         old_name  = content.get("old_name")
         new_name = content.get("new_name")
 
+        # logger.error("Replace template name request")
+
+        codeObj = self.agent.replace_template_name(model,old_name,new_name)
```

Comment:
Nitpick: You have a couple of spots where you do camels, may want to do snakes to be more consistent with the rest of the repo.